### DISCUSSION TITLE
Maximum values

### DISCUSSION THREAD

**Discussion by Q-R-B** [DHI/mikeio#357] [Q&A] [answered]
Hi,
Is there a way to return one timestep with maximum values from a dfsu file (or just from a data array) with many timesteps.
like a way to do some kind of aggregation of the maximum values along the timeserie?

I could not find it anywhere.
thanks for the help

**Comment by jsmariegaard** [comment, marked as answer]
Hi,

This is exactly what the Dataset/DataArray max() method does: 

     > ds = mikeio.read("vol.dfsu")
     > dsmax = ds.max()
     > dsmax.to_dfs("max_vol.dfsu") 

See https://dhi.github.io/mikeio/dataset.html#methods (note the documentation will be updated later today). The same can be done for a DataArray.

**Reply by Q-R-B** [reply]
How can it be that simple, thank you so much for the quick reply!
